packages/terra-image/examples/Index.jsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Markdown from '../../../packages/terra-site/src/Markdown';
+import ReadMe from '../docs/README.md';
+
+const ImageExamples = () => (
+  <div>
+    <Markdown id="readme" src={ReadMe} />
+    <article>
+	  <h2>Fluid Examples</h2>
+
+	  <p>Image width and height are smaller than container. To fill the container, increase the width of the image.</p>
+	  <div className="image-container1">
+	    <img src="http://lorempixel.com/150/150/nature/" alt="rounded" class="terra-Image terra-Image--circle terra-Image--fluid"/>
+	  </div>
+
+	  <p>Image width is larger than the container. Image height is smaller than the container.</p>
+	  <div className="image-container2">
+	    <img src="http://lorempixel.com/150/150/nature/" alt="rounded" class="terra-Image terra-Image--circle terra-Image--fluid"/>
+	  </div>
+
+	  <p>Image width is smaller than the container. Image height is larger than the container.</p>
+	  <div className="image-container3">
+	    <img src="http://lorempixel.com/150/150/nature/" alt="rounded" class="terra-Image terra-Image--circle terra-Image--fluid"/>
+	  </div>
+	</article>
+	<br />
+	<br />
+	<article>
+	  <h2>Non Fluid Examples</h2>
+
+	  <p>Image width and height are smaller than container.</p>
+	  <div className="image-container1">
+	    <img src="http://lorempixel.com/150/150/nature/" alt="rounded" class="terra-Image terra-Image--circle"/>
+	  </div>
+
+	  <p>Image width is larger than the container. Image height is smaller than the container.</p>
+	  <div className="image-container2">
+	    <img src="http://lorempixel.com/150/150/nature/" alt="rounded" class="terra-Image terra-Image--circle"/>
+	  </div>
+
+	  <p>Image width is smaller than the container. Image height is larger than the container.</p>
+	  <div className="image-container3">
+	    <img src="http://lorempixel.com/150/150/nature/" alt="rounded" class="terra-Image terra-Image--circle"/>
+	  </div>
+	</article>
+	<br />
+	<br />
+	/* Styles specific to this demonstration */
+	<style>
```

Comment:
This might need to be reworked. I don't think you can put a style block inside a react component.